.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "*"
+
+  push:
```

Comment:
No value?

Reply:
This can be empty, all push operations will trigger jobs.